Plugins/SimpleFileBrowser/Scripts/FileBrowser.cs

```diff
@@ -2915,8 +2957,7 @@ public static bool SetDefaultFilter( string defaultFilter )
 
 			for( int i = 0; i < Instance.filters.Count; i++ )
 			{
-				HashSet<string> extensions = Instance.filters[i].extensionsSet;
-				if( extensions != null && extensions.Contains( defaultFilter ) )
+				if( Instance.filters[i].isValidExtension( defaultFilter ) )
```

Comment:
We can change it as follows and remove the isValidExtension property: `Instance.filters[i].MatchesExtension( defaultFilter, defaultFilter.LastIndexOf( '.' ) != 0 )`. Even better, the value of `defaultFilter.LastIndexOf( '.' ) != 0` can be cached in a `bool defaultFilterHasMultipleSuffixes` variable before the for-loop.